src/pretalx/cfp/views/wizard.py

```diff
@@ -143,6 +136,23 @@ def done(self, form_list, form_dict, **kwargs):
                     answer.answer = value
                 answer.save()
 
+        try:
+            sub.event.ack_template.to_mail(
+                user=user, event=self.request.event, context=template_context_from_submission(sub),
+                skip_queue=True, locale=user.locale
+            )
+            if self.request.event.settings.mail_on_new_submission:
+                MailTemplate(
+                    event=self.request.event,
+                    subject=_('[{event}] New submission!').format(event=self.request.event.slug),
+                    text=self.request.event.settings.mail_text_new_submission,
+                ).to_mail(
+                    user=user, event=self.request.event, context=template_context_from_submission(sub),
+                    skip_queue=True, locale=self.request.event.locale
+                )
+        except SendMailException:
```

Comment:
Split this `try`, in case there is a problem specific to one template or mail address, we allow the other one to go through. Probably unlikely, but worth a shot, I think.